src-bindings/vscode/vscode.mli

```diff
@@ -782,6 +782,22 @@ module CancellationToken : sig
     -> t
 end
 
+module CustomDocument : sig
+  module type T = sig
+    include Js.T
+
+    val uri : t -> Uri.t
+
+    val dispose : t -> unit
+  end
+
+  include Js.T
```

Comment:
This line is unnecessary since `T` already includes `Js.T`.

Reply:
Good catch, thanks :)